### ISSUE TITLE
Release 1.1

### ISSUE THREAD

**Issue by blink1073** [jupyterlab/jupyterlab#7074]
- [x] Release the RC today at 1200 CT
- [ ] Update change log
- [ ] Audit existing 1.1 issues and PRs and move to appropriate milestones
- [ ] Release 1.1 Final on 28 August

**Comment by blink1073** [comment]
$ loghub jupyterlab/jupyterlab -m 1.1 -t $GITHUB_TOKEN --template scripts/release_template.txt 

```rst
* Correct `quit_button` Test (`#7071 <https://github.com/jupyterlab/jupyterlab/pull/7071>`__, `#7070 <https://github.com/jupyterlab/jupyterlab/issues/7070>`__)
* Add A Note To Check Out A Branch Before Doing A Release. (`#7066 <https://github.com/jupyterlab/jupyterlab/pull/7066>`__)
* Increase The Search Debounce From 100ms To 500ms (`#7034 <https://github.com/jupyterlab/jupyterlab/pull/7034>`__)
* Complete Docstring For Iexecuterequestmsg (`#7028 <https://github.com/jupyterlab/jupyterlab/pull/7028>`__)
* Restores Silent Failure Behavior When A Filetype Has No Iconclass (`#7027 <https://github.com/jupyterlab/jupyterlab/pull/7027>`__)
* Fix Vega Downloads And Download Urls In General (`#7022 <https://github.com/jupyterlab/jupyterlab/pull/7022>`__, `#7017 <https://github.com/jupyterlab/jupyterlab/issues/7017>`__)
* Restore Behavior Of The "raises-Exception" Cell Tag (`#7020 <https://github.com/jupyterlab/jupyterlab/pull/7020>`__, `#7015 <https://github.com/jupyterlab/jupyterlab/issues/7015>`__)
* Properly Escape Template Variables (`#7016 <https://github.com/jupyterlab/jupyterlab/pull/7016>`__, `#7024 <https://github.com/jupyterlab/jupyterlab/issues/7024>`__)
* Do Not Fail On Deduplicate And Ignore Optional (`#7013 <https://github.com/jupyterlab/jupyterlab/pull/7013>`__)
* Fix A Typo In The Issues Reporting Documentation (Issue.rst) (`#6993 <https://github.com/jupyterlab/jupyterlab/pull/6993>`__)
* Make Dependencies Of Vdom Extension Optional  (`#6980 <https://github.com/jupyterlab/jupyterlab/pull/6980>`__)
* Make Dependencies Of Csv Viewer Extension Optional (`#6978 <https://github.com/jupyterlab/jupyterlab/pull/6978>`__)
* Add Docs To Help Users Diagnose Issues Before Creating Them (`#6971 <https://github.com/jupyterlab/jupyterlab/pull/6971>`__)
* Remove Lockbot Message, Re-Engage Lockbot (`#6963 <https://github.com/jupyterlab/jupyterlab/pull/6963>`__)
* Add @types/jest To Extension Testing Doc (`#6961 <https://github.com/jupyterlab/jupyterlab/pull/6961>`__)
* Pause Lockbot (`#6953 <https://github.com/jupyterlab/jupyterlab/pull/6953>`__)
* Fix Comm_info_request Content To Conform To Spec In A Backwards-Compatible Way (`#6949 <https://github.com/jupyterlab/jupyterlab/pull/6949>`__, `#6947 <https://github.com/jupyterlab/jupyterlab/issues/6947>`__)
* Add Lockbot Configuration To Lock Inactive Closed Issues/prs After 30 Days (`#6943 <https://github.com/jupyterlab/jupyterlab/pull/6943>`__)
* Adds Yarn Package Resolution To Build (`#6938 <https://github.com/jupyterlab/jupyterlab/pull/6938>`__)
* Add Display:flex To .jp-Inspector-Default-Content (`#6935 <https://github.com/jupyterlab/jupyterlab/pull/6935>`__)
* Fix For Contextual Help Layout (`#6933 <https://github.com/jupyterlab/jupyterlab/pull/6933>`__)
* Attempted Fix For Minor Jupyterhub Compatibility Issue. (`#6931 <https://github.com/jupyterlab/jupyterlab/pull/6931>`__)
* Make Handling Comm Messages Optional In A Kernel Connection. (`#6929 <https://github.com/jupyterlab/jupyterlab/pull/6929>`__)
* Install Multiple Versions Of Same Extension (`#6857 <https://github.com/jupyterlab/jupyterlab/pull/6857>`__)
* Fix/slow Search (`#6824 <https://github.com/jupyterlab/jupyterlab/pull/6824>`__, `#6756 <https://github.com/jupyterlab/jupyterlab/issues/6756>`__)
* Remove Focus On Esc In Console (`#6822 <https://github.com/jupyterlab/jupyterlab/pull/6822>`__)
* Reevaluate Output Renderer On Some Changes (`#6763 <https://github.com/jupyterlab/jupyterlab/pull/6763>`__, `#6762 <https://github.com/jupyterlab/jupyterlab/issues/6762>`__)
* Added Debug Messages (`#6704 <https://github.com/jupyterlab/jupyterlab/pull/6704>`__)
* Fix Downloads In Chrome With Xsrf Token (`#6686 <https://github.com/jupyterlab/jupyterlab/pull/6686>`__)
* Clarifies Workflow To Export As Reveal.js Slides (`#6472 <https://github.com/jupyterlab/jupyterlab/pull/6472>`__, `#5018 <https://github.com/jupyterlab/jupyterlab/issues/5018>`__)
* Expose Icon Svg To Theme Css (`#6034 <https://github.com/jupyterlab/jupyterlab/pull/6034>`__)
* Profile With Lighthouse (`#5870 <https://github.com/jupyterlab/jupyterlab/pull/5870>`__, `#4825 <https://github.com/jupyterlab/jupyterlab/issues/4825>`__)
```

**Comment by jasongrout** [comment]
1.1 is now released on pypi and conda-forge.
